Frameworks/Foundation/NSProgress.mm

```diff
@@ -208,7 +209,9 @@ - (void)resignCurrent {
         // If you don’t create any child progress objects between the calls to becomeCurrentWithPendingUnitCount: and resignCurrent,
         // the "parent" progress automatically updates its completedUnitCount by adding the pending units.
         if (!currentProgressStack->top().childCreated) {
```

Comment:
it's not, it returns the wrapper

---
In reply to: [95282701](https://github.com/Microsoft/WinObjC/pull/1663#discussion_r95282701) [](ancestors = 95282701)